<!-- Launch in the camera namespace containing "image_raw" and "camera_info" -->
<launch>

  <arg name="manager" /> <!-- Must be globally qualified -->
  <arg name="respawn" default="false" />
  <!-- TODO Arguments for debayer, interpolation methods? -->

  <arg     if="$(arg respawn)" name="bond" value="" />
  <arg unless="$(arg respawn)" name="bond" value="--no-bond" />

  <!-- Monochrome rectified image -->
  <node pkg="nodelet" type="nodelet" name="process_image"
        args="load blob_tracker/process_image $(arg manager) $(arg bond)"
	    respawn="$(arg respawn)" />

</launch>
